### ISSUE TITLE
ModuleNotFoundError: No module named 'requests_cache'

### ISSUE THREAD

**Issue by sanjeevanahilan** [parkerhancock/patent_client#118]
Hi, I think there may be a newly introduced bug as I upgraded patent_client from 3.2.8 to 3.1.0 and got the following error:

```
    from patent_client import Patent
  File "/Users/sanjeevan/miniforge3/envs/solve/lib/python3.10/site-packages/patent_client/_init_.py", line 57, in <module>
    from patent_client.uspto.assignment.model import Assignment  # isort:skip
  File "/Users/sanjeevan/miniforge3/envs/solve/lib/python3.10/site-packages/patent_client/uspto/_init_.py", line 1, in <module>
    from .assignment import Assignment
  File "/Users/sanjeevan/miniforge3/envs/solve/lib/python3.10/site-packages/patent_client/uspto/assignment/_init_.py", line 1, in <module>
    from .model import Assignment
  File "/Users/sanjeevan/miniforge3/envs/solve/lib/python3.10/site-packages/patent_client/uspto/assignment/model.py", line 9, in <module>
    from patent_client import session
  File "/Users/sanjeevan/miniforge3/envs/solve/lib/python3.10/site-packages/patent_client/session.py", line 4, in <module>
    import requests_cache
```

For now it works when I specify 3.2.8

**Comment by parkerhancock** [comment]
The quick fix is to install requests_cache. I'll push an update this afternoon to formally remove that problem. Thanks for flagging!

**Comment by sanjeevanahilan** [comment]
Thanks!

**Comment by parkerhancock** [comment]
Fixed in #119, please update to v4.0.0!
